Labs/ARP Tabelle/README.md

```diff
@@ -43,14 +43,14 @@ Der Befehl `arp -n` hat nun einen unvollständigen Eintrag für diese IP Adresse
 4.  Wiederholen Sie Schritt 3 mit einem Gerät in Ihrem Netzwerk (bspw. Smartphone, TV, Musikanlage, NAS etc.):
 
 
-    ping -c 3 192.168.105.53
-    PING 192.168.105.53 (192.168.105.53) 56(84) bytes of data.
-    64 bytes from 192.168.105.53: icmp_seq=1 ttl=64 time=4.36 ms
-    64 bytes from 192.168.105.53: icmp_seq=2 ttl=64 time=2.83 ms
-    64 bytes from 192.168.105.53: icmp_seq=3 ttl=64 time=1.23 ms
-
-    --- 192.168.105.53 ping statistics ---
-    3 packets transmitted, 3 received, 0% packet loss, time 2003ms
+    ping -c 3 192.168.105.53  
+    PING 192.168.105.53 (192.168.105.53) 56(84) bytes of data.  
+    64 bytes from 192.168.105.53: icmp_seq=1 ttl=64 time=4.36 ms  
+    64 bytes from 192.168.105.53: icmp_seq=2 ttl=64 time=2.83 ms  
+    64 bytes from 192.168.105.53: icmp_seq=3 ttl=64 time=1.23 ms  
+  
+    --- 192.168.105.53 ping statistics ---  
+    3 packets transmitted, 3 received, 0% packet loss, time 2003ms  
     rtt min/avg/max/mdev = 1.234/2.805/4.355/1.274 ms
```

Comment:
I think four more spaces at the beginning of the line should fix this formatting issue:

```
        ping -c 3 192.168.105.53  
        PING 192.168.105.53 (192.168.105.53) 56(84) bytes of data.  
        64 bytes from 192.168.105.53: icmp_seq=1 ttl=64 time=4.36 ms  
        64 bytes from 192.168.105.53: icmp_seq=2 ttl=64 time=2.83 ms  
        64 bytes from 192.168.105.53: icmp_seq=3 ttl=64 time=1.23 ms  
  
        --- 192.168.105.53 ping statistics ---  
        3 packets transmitted, 3 received, 0% packet loss, time 2003ms  
        rtt min/avg/max/mdev = 1.234/2.805/4.355/1.274 ms
```

This also applies to Task 3. The two space characters at the end of the line are obsolete.